engine/class_modules/paladin/sc_paladin.cpp

```diff
@@ -3490,7 +3510,12 @@ struct paladin_module_t : public module_t
 
     // 9.0 Paladin Night Fae
 
-    // Only create these if the player is a Night Fae Paladin or sets the option to get the buff.
+    // Only create these if the player is a Night Fae Paladin or sets the option to get the 
+    // 
+    // 
+    // 
+    // 
+    // 
```

Comment:
```suggestion
    // Only create these if the player is a Night Fae Paladin or sets the option to get the buff.
```
undo weird changes to comments